test/tests/utils/spline_interpolation/tests

```diff
@@ -1,8 +1,13 @@
 [Tests]
-  [./test]
+  [./regular_spline]
     type = 'Exodiff'
     input = 'spline_interpolation.i'
     exodiff = 'spline_interpolation_out.e'
   [../]
+  [./bicubic_spline]
+    type = 'Exodiff'
+    input = 'bicubic_spline_interpolation.i'
+    exodiff = 'bicubic_spline_interpolation_out.e'
+    custom_cmp = 'bicubic_spline_interpolation.cmp'
+  [../]
 []
-  
```

Comment:
I shouldn't have said it was a github error; here's the file in whitespace-mode before I modified it:

![bad_whitespace](https://cloud.githubusercontent.com/assets/10248304/20727612/cf21e26c-b63f-11e6-8462-e7bf0fe6bdbd.png)

so github wasn't lying or producing an error of any kind. It was just noting that I **removed** a line at the end of the file with whitespace that didn't have a newline character at the end. So instead of being bad, I removed badness :-)